dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaServicesTaskTest.java

```diff
@@ -16,140 +16,108 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
 import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
 import com.google.common.io.ByteSink;
 import com.google.common.io.CharSink;
 import com.google.edwmigration.dumper.application.dumper.MetadataDumperUsageException;
 import com.google.edwmigration.dumper.application.dumper.connector.cloudera.manager.ClouderaManagerHandle.ClouderaClusterDTO;
 import com.google.edwmigration.dumper.application.dumper.task.TaskRunContext;
-import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.Writer;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.HashSet;
-import java.util.Set;
-import org.apache.http.HttpEntity;
-import org.apache.http.client.methods.CloseableHttpResponse;
-import org.apache.http.client.methods.HttpGet;
-import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.junit.AfterClass;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
 @RunWith(MockitoJUnitRunner.class)
 public class ClouderaServicesTaskTest {
+  private static WireMockServer server;
 
   private final ClouderaServicesTask task = new ClouderaServicesTask();
 
   private ClouderaManagerHandle handle;
 
-  private String servicesJson;
   @Mock private TaskRunContext context;
   @Mock private ByteSink sink;
 
   @Mock private Writer writer;
   @Mock private CharSink charSink;
 
-  @Mock private CloseableHttpClient httpClient;
-  private URI uri;
+  @BeforeClass
+  public static void beforeClass() throws Exception {
+    server = new WireMockServer(WireMockConfiguration.wireMockConfig().dynamicPort());
+    server.start();
+  }
+
+  @AfterClass
+  public static void afterClass() throws Exception {
+    server.stop();
+  }
 
   @Before
   public void setUp() throws Exception {
-    servicesJson = readFileAsString("/cloudera/manager/cluster-status.json");
-    uri = URI.create("http://localhost/api");
-    handle = new ClouderaManagerHandle(uri, httpClient);
+    server.resetAll();
+
+    URI uri = URI.create(server.baseUrl() + "/api/vTest");
+    handle = new ClouderaManagerHandle(uri, HttpClients.createDefault());
 
     when(sink.asCharSink(eq(StandardCharsets.UTF_8))).thenReturn(charSink);
     when(charSink.openBufferedStream()).thenReturn(writer);
   }
 
   @Test
   public void clustersExists_do_writes_success() throws Exception {
+    String servicesJson = readFileAsString("/cloudera/manager/cluster-status.json");
```

Comment:
Do we need all the data in this file for this test?

Reply:
No, we don't. The example of random json is below. `.willReturn(okJson("{\"some\": \n \"json\" \r}")))`. I just want to have an example of response at least in test.
In practice it can be any valid json.